cookbook/core/control_flow/subworkflows.py

```diff
@@ -100,6 +87,90 @@ def nested_parent_wf(a: int) -> Tuple[int, str, str, str]:
 
 
 # %%
-# You can execute the nested workflows locally as well
+# You can execute the nested workflows locally as well.
 if __name__ == "__main__":
     print(f"Running nested_parent_wf(a=3) {nested_parent_wf(a=3)}")
+
+
+# %%
+# External Workflow
+# -----------------
+#
+# When launch plans are used within a workflow to launch an execution of a previously defined workflow, a new
+# external execution is launched, with a separate execution ID and can be observed as a distinct entity in
+# FlyteConsole/FlyteCTL etc. 
+#
+# Moreover, the context is not shared, which means they may have separate parallelism constraints.
+# We refer to these externalized invocations of a workflow using launch plans from a parent workflow as ``External Workflows``.
+#
+# If your deployment uses :ref:`multicluster-setup <Using Multiple Kubernetes Clusters>`, then external workflows may allow you to distribute the workload of a workflow to multiple clusters.
+# Here is an example demonstrating external workflows:
+
+# %%
+import typing
+from typing import Tuple
+from flytekit import task, workflow
+from flytekit import conditional, task, workflow, LaunchPlan
+
+# %%
+# We define a task that adds and subtracts two numbers when a condition is fulfilled.
```

Comment:
Since the two tasks are simple enough to understand, can we say, "We define two tasks to call from within a workflow."?